wip/services/exec/service.go

```diff
@@ -106,7 +105,7 @@ func (s *ProviderService) Create(ctx context.Context, project string, creator st
 		Str(types.ExecIDCtxKey, execID).
 		Msgf("Created new exec with image %q", image)
 
-	exec := types.Exec{
+	unscheduledExec := types.Exec{
```

Comment:
I don't think we need to do this. Whether it's unscheduled or not is not know here. Simpler names are better too.

Reply:
I think it’s implicit that it’s not scheduled until an informer is watching the Exec to be scheduled, but sure